staging/src/k8s.io/kubectl/pkg/kuberc/kuberc.go

```diff
@@ -445,3 +448,51 @@ func validate(plugin *config.Preference) error {
 
 	return nil
 }
+
+// loadAliasesForHelpDisplay loads all aliases from the kuberc file and registers them
+// in the command hierarchy for help display. This is called when kubectl is run without arguments.
+func (p *Preferences) loadAliasesForHelpDisplay(rootCmd *cobra.Command, errOut io.Writer) {
+	kuberc, err := p.getPreferencesFunc("", errOut)
```

Comment:
This won't work with `kubectl --kuberc mykuberc -h`. In general, I'd prefer to re-use as much as possible of the existing logic, rather than have entirely separate methods just for the help display. Having two paths for somewhat similar purpose introduces the risk of misalignment between these two paths. So I'd rather have paths which share code as much as possible.